geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/audit/AuditedEvent.scala

```diff
@@ -0,0 +1,63 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.utils.audit
+
+import java.io.Closeable
+
+import com.google.gson.{Gson, GsonBuilder}
+import com.typesafe.scalalogging.LazyLogging
+import org.joda.time.Interval
+
+import scala.reflect.ClassTag
+
+trait AuditedEvent {
```

Comment:
javadoc on traits audited event and deletable event here if you have time to explain high level what they are for